languages/AspireUpdate-nl_NL.po

```diff
@@ -0,0 +1,129 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: AspireUpdate\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2024-11-04 06:19+0000\n"
+"PO-Revision-Date: 2024-11-05 14:09+0100\n"
+"Last-Translator: \n"
+"Language-Team: Dutch (Netherlands)\n"
+"Language: nl_NL\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 3.5\n"
+"X-Loco-Version: 2.6.11; wp-6.6.2\n"
+"X-Domain: AspireUpdate\n"
+
+#: includes/class-admin-settings.php:313
+msgid "API Configuration"
+msgstr "API Configuratie"
+
+#: includes/class-admin-settings.php:378
+msgid "API Debug Configuration"
+msgstr "API Debug Configuratie"
+
+#: includes/class-admin-settings.php:337
+msgid "API Host"
+msgstr "API Host"
+
+#: includes/class-admin-settings.php:364
+msgid "API Key"
+msgstr "API Sleutel"
+
+#. Author of the plugin
+msgid "AspirePress"
+msgstr "AspirePress"
+
+#. Name of the plugin
+msgid "AspireUpdate"
+msgstr "AspireUpdate"
+
+#: includes/class-admin-settings.php:272
+msgid "AspireUpdate Settings"
+msgstr "AspireUpdate Instellingen"
+
+#: includes/class-admin-settings.php:422
+msgid "Disable SSL Verification"
+msgstr "SSL Verificatie uitschakelen"
+
+#: includes/class-admin-settings.php:431
+msgid "Disables the verification of SSL to allow local testing."
+msgstr "Schakel SSL verificatie uit om lokaal testen mogelijk te maken."
+
+#: includes/class-admin-settings.php:324
+msgid "Enable AspireUpdate API Rewrites"
+msgstr "AspireUpdate API herschrijfregels inschakelen"
+
+#: includes/class-admin-settings.php:389
+msgid "Enable Debug Mode"
+msgstr "Debug Modus inschakelen"
+
+#: includes/class-admin-settings.php:403
+msgid "Enable Debug Type"
+msgstr "Debug Type inschakelen"
+
+#: includes/class-admin-settings.php:397
+msgid "Enables debug mode for the plugin."
+msgstr "Schakelt debug modus in voor de plugin."
+
+#: includes/class-admin-settings.php:494
+msgid "Generate API Key"
+msgstr "Genereer API Sleutel"
+
+#. URI of the plugin
+msgid "https://aspirepress.org/"
+msgstr "https://aspirepress.org/"
+
+#. Author URI of the plugin
+msgid "https://docs.aspirepress.org/aspireupdate/"
+msgstr "https://docs.aspirepress.org/aspireupdate/"
+
+#: includes/class-admin-settings.php:355
+msgid "Other"
+msgstr "Overig"
+
+#: includes/class-admin-settings.php:416
+msgid ""
+"Outputs the request URL and headers / response headers and body / string "
+"that is being rewritten."
+msgstr ""
+"Toon de aanvraag URL en headers / antwoord headers en body / tekst die "
+"wordt herschreven."
+
+#: includes/class-admin-settings.php:372
+msgid "Provides an API key for repositories that may require authentication."
+msgstr ""
```

Comment:
The String brackets seem wrong, let's fix

Reply:
@namithj What is wrong with it? This is exactly how Poedit outputs the file. And have a look at the other translations, they are the same. When a translation (or source string) is too long this is the way they are added to the po-file